frmts/dted/dted_api.c

```diff
@@ -525,13 +525,15 @@ int DTEDReadPoint(DTEDInfo *psDInfo, int nXOff, int nYOff, GInt16 *panVal)
     if (pabyData[0] & 0x80)
     {
         *panVal *= -1;
+        const char* pszAssume = CPLGetConfigOption("DTED_ASSUME_CONFORMANT", "NO");
+        const bool bAssumeConformant = CPLTestBool(pszAssume);
```

Comment:
reading config option is time expensive:

```suggestion
        static const bool bAssumeConformant = CPLTestBool(
            CPLGetConfigOption("DTED_ASSUME_CONFORMANT", "NO"));
```